pkg/server/server.go

```diff
@@ -99,65 +110,27 @@ func (s *AdmissionWebhookServer) listenAndServe(clients *clients.Clients, handle
 		// Sleep here to make sure server is listening and all caches are primed
 		time.Sleep(15 * time.Second)
 
```

Comment:
it is not related to this PR, but `sleep 15 to wait http server is ready` is tricky, check a flag or use a channel to sync will be a further enhancement

Reply:
Yes, I have filed an issue to track it.